components/button/test/floating-buttons.html

```diff
@@ -0,0 +1,121 @@
+<!doctype html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>d2l-floating-buttons basic tests</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script src="/node_modules/mocha/mocha.js"></script>
+		<script src="/node_modules/chai/chai.js"></script>
+		<script src="/node_modules/@polymer/test-fixture/test-fixture.js"></script>
+		<script src="/node_modules/wct-mocha/wct-mocha.js"></script>
+		<script type="module" src="../button.js"></script>
+		<script type="module" src="../floating-buttons.js"></script>
+	</head>
+	<body>
+		<test-fixture id="floating-buttons-fixture">
+			<template>
+				<d2l-floating-buttons>
+					<d2l-button id="btn-make-more">Make more Beer, on a phone!</d2l-button>
+				</d2l-floating-buttons>
+			</template>
+		</test-fixture>
+
+		<script type="module">
+			import { runAxe } from '../../../tools/a11y-test-helper.js';
+			const isMobileHeight = Math.max(document.documentElement.clientHeight, window.innerHeight || 0) <= 500;
```

Comment:
this used to set a specific width,  I feel that didn't make sense as the component only uses the height when determining the mobile setting. I've changed this test to work with height as well.

Reply:
I think a lot (perhaps all except the aXe) of these tests could be covered by visual-diff.  In some cases, it may be easier to reload the page in order to "reset" the state however.